cmake/modules/AddSwift.cmake

```diff
@@ -597,6 +599,112 @@ function(add_swift_host_library name)
   endif()
 endfunction()
 
+# Add a module of libswift
```

Comment:
Can you add to all of the comments here that this is a temporary addition to AddSwift.cmake until fixes in the swift-driver are available on macOS host toolchains? That way it is 100% clear in the source that this is going away once what I mentioned happens.